### PR TITLE
Forgot to bind

### PR DESCRIPTION
**Pull request by juliangarritano**

### CODE REVIEW

File: src/js/models/appUser.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ var AppUser = module.exports = BaseModel.extend({
             return Backbone.Model.prototype.save.apply(this, _.toArray(arguments)).then(function() {
                 this._lastPropertyValues = this.pick(AppUser.EDITABLE_PROPERTIES);
                 return arguments;
-            });
+            }.bind(this));
```
Comment (lemieux) [inline src/js/models/appUser.js]: use `_.bind` instead for compatibility with old crumbling browsers.

Comment (juliangarritano) [inline src/js/models/appUser.js]: We're not supporting old crumbling browsers though...
I hate the way _.bind looks.
If we have to go through the codebase to fix them all, I volunteer.

Comment (lemieux) [inline src/js/models/appUser.js]: I hate it too, but that's a small thing to add support for those browsers. And `_(function(){}).bind(this)` looks almost the same ;)

Comment (lemieux) [inline src/js/models/appUser.js]: Hmm nevermind... fn.bind is supported by IE9 and up.

Comment (juliangarritano) [inline src/js/models/appUser.js]: Yeah that's what I meant when I said we aren't supporting crumbling browsers. I thought you meant IE 8 :tongue:

**Comment by lemieux** [comment]
Delete your branch though.

File: src/js/models/appUser.js (hunk 1 above)
Comment (dannytranlx) [inline src/js/models/appUser.js]: The issue was with PhantomJS 1.x which runs our client-side tests. Might need to update to PhantomJS 2.x and all the related Grunt packges.

**Pull request merged**